spec.html

```diff
@@ -40161,6 +40161,14 @@ <h1>`Promise.any` Reject Element Functions</h1>
         </emu-clause>
       </emu-clause>
 
+      <emu-clause id="sec-promise.anysettled">
+        <h1>Promise.anySettled ( _iterable_ )</h1>
+        <emu-note>
+          <p>The *"anySettled"* property is an alias for the `Promise.race` and was added for consistency in the naming of promise combinators.</p>
+        </emu-note>
+        <p>The initial value of the *"anySettled"* property is the same function object as the initial value of the `Promise.race` property.</p>
```

Comment:
this means that the `.name` of the function would be `.race` - usually the `.name` matches the primary function, not the alias.

Reply:
Thanks! I've added paragraph about `.name` property